test/sparse/cholmod.jl

```diff
@@ -1,11 +1,13 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-srand(123)
-
 using Base.SparseArrays.CHOLMOD
 using DelimitedFiles
+using Test
```

Comment:
Is it still necessary to import `Test` ?

Reply:
Not strictly but it makes it possible to run the file alone and it doesn't hurt.